packages/zwave-js/src/lib/commandclass/CommandClass.ts

```diff
@@ -375,12 +375,18 @@ export class CommandClass {
 					reason = e.context;
 				}
 
-				return new InvalidCC(driver, {
+				const invalidcc = new InvalidCC(driver, {
```

Comment:
To keep the convention of this codebase, let's call this `ret`

Reply:
Sure - done.